src/docs/product/alerts/create-alerts/high-priority-alerts.mdx

```diff
@@ -0,0 +1,24 @@
+# Priority-based Alerts in Sentry
+
+Sentry is rolling out a set of updates to make alerts more actionable. Rather than defaulting to sending alerts on every new issue, we’ll only notify on issues that pass a priority threshold based on historical actions on similar issues.
+
+To start, we’ll modify the default alerts with Sentry projects with a new condition that filters alerts to only high-priority issues. Each new issue will have a score calculated for the importance of the first event. If that first event is deemed to be high severity or has a fatal log level, we'll alert you. If the first event is low severity, we'll only alert you if the issue is forecasted to worsen over time.
+
+In summary, the high priority alert condition will notify you if an issue is:
+
+- Similar to issues that have been viewed & responded to quickly in the past, or likely to be actionable (read on for more details)
+- Escalating in volume: for new issues, we’ll compare the hourly event rate to the 95th percentile hourly event rate for new issues in the project
```

Comment:
not sure how specific we want to be with the semantics but the 95th percentile hourly event rate can include non-new issues; the condition to be included is to have more than 1 event in the past week and the issue itself being older than an hour

Reply:
good call, updated this a bit!